xpra/server/window/window_video_source.py

```diff
@@ -723,6 +727,8 @@ def matches_video_subregion(self, width, height):
         return vr
 
     def subregion_is_video(self):
+        if WINDOW_FOCUS_VIDEO_ENCODER and self.has_focus:
+            return True
```

Comment:
This feels like the wrong place for it.
What is this meant to do?

Reply:
probably not ideal, yes - here again the intent was to short-circuit any video region detection for a window and always use nvenc for the active/focused window